Add missing meta descriptions (#180)
mongodb/docs-rust: source/quick-reference.txt

Quick Reference
===============

.. meta::

>    :description: Explore examples of common MongoDB tasks using the Rust driver, including asynchronous and synchronous runtimes.

Reviewer: ```suggestion
   :description: Explore examples of common tasks using the MongoDB Rust Driver, using asynchronous or synchronous runtimes.
```
